cpp/ps0000/p0.cpp

```diff
@@ -1,9 +1,10 @@
-#include <iostream>
+#include <iostream> // 2/2
 
 using namespace std;
 
-int main()
+int main() // 1/1
 {
-    cout << "Hello world " << endl;
-return 0;
+    cout << "Hello, World!" << endl; // 3/4
+    return 0; // 1/1
 }
+// great work 😎 9/10 ⭐

```

Comment:
- [x] most of code errors and bugs are fixed
- [x] file is in the right path
- [x] +1 bonus marks for your first commit